src/index.spec.ts

```diff
@@ -7,6 +7,159 @@ import {
   createProduct 
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
-});
+describe('isInteger',()=>{
```

Comment:
Try to configure your linter and formatter, to automatically add blank spaces, like after the comma:
`() => {`